### DISCUSSION TITLE
blog/gpt4-generating-code

### DISCUSSION THREAD

**Discussion by giscus** [Marviel/lukebechtel-web#1] [General]
# blog/gpt4-generating-code

When the machines make machines, we're in a new world.

http://localhost:3002/blog/gpt4-generating-code

**Comment by Marviel** [comment]
Hello World

**Comment by pauladis** [comment]
not gonna lie, that's awesome

**Comment by Yukinyaa** [comment]
This is exciting and terrifying at the same time.
I got to play with LLM and now ChatGPT and bing chat is literally on my side whenever I work.
Now after reading this, I really might have to check GPT4 out

**Reply by Marviel** [reply]
Definitely recommend ChatGPT Plus!
